core/cmd/client.go

```diff
@@ -266,9 +269,10 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg chainlink.G
 			if err != nil {
 				return nil, errors.Wrap(err, "failed to marshal Solana configs")
 			}
+			envConfig := plugins.NewEnvConfig(cfg.LogLevel(), cfg.JSONConsole(), cfg.LogUnixTimestamps(), portManager.Register(cmdName))
```

Comment:
This `cmdName` is unique for now, but only because this is a special case singleton plugin (and only temporarily). WDYT about using the logger/service names? They are already unique within the app, and it would keep things forward compatible.